ledger/participant-integration-api/src/main/scala/platform/store/backend/h2/H2StorageBackend.scala

```diff
@@ -228,4 +229,15 @@ private[backend] object H2StorageBackend
     throw new UnsupportedOperationException("db level locks are not supported for H2")
 
   override def dbLockSupported: Boolean = false
+
+  // Migration from mutable schema is not supported for H2
+  override def validatePruningOffsetAgainstMigration(
+      pruneUpToInclusive: Offset,
+      pruneAllDivulgedContracts: Boolean,
+      connection: Connection,
+  ): Unit = {
+    discard(pruneUpToInclusive)
```

Comment:
if you don't do these discards, it complains due to not used params? even f the override is there? :thinking:

Reply:
Ha, it seems not need, right. Probably because of the beauties brought by inheritance-constrained designs :trollface: Removed